sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -386,6 +394,7 @@ def test_series_tests(self):
             ],
             'pandas.core.series.Series.fillna': [
                 "df.fillna(method='ffill')",
+                'df.fillna(method="ffill")',
```

Comment:
duplicate?

Reply:
Pandas changed from single to double quotes in 1.3.0, so we need to add this extra line in.

Reply:
Gotcha. I missed the quote difference when I was first reading.